cognite/client/data_classes/aggregations.py

```diff
@@ -215,8 +210,8 @@ def dump(self, camel_case: bool = True) -> dict[str, Any]:
         return {"start": self.start, "count": self.count}
 
 
-class Buckets(UserList):
-    def __init__(self, items: Collection[Any]) -> None:
+class Buckets(UserList, MutableSequence[Bucket]):
+    def __init__(self, items: Collection[dict | Bucket]) -> None:
         super().__init__([Bucket(**bucket) if isinstance(bucket, dict) else bucket for bucket in items])
```

Comment:
Can `Bucket` ever be expected to get new fields? (Looking at the **)

Reply:
I would say yes - good catch!